test/Microsoft.AspNetCore.Mvc.Core.Test/PhysicalFileResultTest.cs

```diff
@@ -204,6 +206,23 @@ public TestPhysicalFileResult(string filePath, string contentType)
             {
             }
 
+            public override Task ExecuteResultAsync(ActionContext context)
+            {
+                var executor = context.HttpContext.RequestServices.GetRequiredService<TestPhysicalFileResultExecutor>();
+                executor.IsAscii = IsAscii;
+                return executor.ExecuteAsync(context, this);
+            }
+
+            public bool IsAscii { get; set; } = false;
```

Comment:
Where in ASP.NET Core do you see us following that convention? In any case, this is test code. There's no reason to change the existing test results and every reason to match the existing name in the new executors.